ocpmodels/trainers/ocp_trainer.py

```diff
@@ -338,15 +337,15 @@ def _compute_loss(self, out, batch):
                 target = target.view(batch_size, -1)
 
             mult = loss_info["coefficient"]
-            loss.append(
-                mult
-                * loss_info["fn"](
-                    pred,
-                    target,
-                    natoms=natoms,
-                    batch_size=batch_size,
-                )
+            loss_ = mult * loss_info["fn"](
+                pred,
+                target,
+                natoms=natoms,
+                batch_size=batch_size,
             )
+            if self.scaler:
```

Comment:
scaling here means that the scaling is done inside [the autocast context](https://github.com/Open-Catalyst-Project/ocp/blob/9108a87ce383b2982c24eff4178632f01fecb63e/ocpmodels/trainers/ocp_trainer.py#L158) in the `train` method. From the docs, I only see scaling done outside the context, so I'm not sure if this could lead to other issues downstream.

One solution may be return the list of losses and scale in `train` outside the autocast context and then take the sum of losses (would also need to make sure to sum the losses anywhere else that uses `_compute_loss` for example `validate`. Another solution may be to keep this as is and disable autocast in here for scaling.